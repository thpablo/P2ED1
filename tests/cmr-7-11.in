7 11
r
***********
*         *
********* *
*M*   *   *
* *** * * *
*          
***********
