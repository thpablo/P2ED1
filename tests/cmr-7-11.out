11
***********
*         *
********* *
*M*   *   *
*o*** * * *
*oooooooooo
***********
